### ISSUE TITLE
Downloader no longer works

### ISSUE THREAD

**Issue by funnyandcreativeusername** [Sibusten/derpibooru-downloader#59]
putting in tags no longer downloads anything. is there something preventing things from being downloaded?

**Comment by 4lcane** [comment]
Same here

**Comment by AndrewTru** [comment]
Well if the downloader uses API calls and the site URL has www, they deprecated that URL.

I don't know the new URL, but website.com works in my browser.

So best shot the new URL is Derpibooru.com?
Maybe they changed your API key?

**Comment by Twi-Hard** [comment]
https://derpibooru.org/forums/meta/topics/deprecation-of-the-www-subdomains

**Comment by 4lcane** [comment]
No, it's not the API. I'm guessing it's the deprecation of the www in the URL. All you'd need to do is change the source code and that COULD fix the issue, but I've already moved on to derpi-dl. It's command line based, but works just about the same.

**Comment by Astrocities** [comment]
Works for me on v2.1.0 after replacing `https://www.derpibooru.org` with `https://derpibooru.org/` in Booru URL.

![2025-01-08 02_09_00-Derpibooru Downloader v2 1 0](https://github.com/user-attachments/assets/2149935d-cd75-40b1-9a9b-d49807b93a55)

**Comment by speedingcheetah** [comment]
Its working, but, it is not downloading ALL available user images.  It gives too many requests error for several, but i manually go in and download those. Even adding those, it is missing hundreds of images. example. User has 3,410 images, this only downloads 3,236.

**Comment by 4lcane** [comment]
For whatever reason, I didn't see the "Booru URL" box, lol. By removing the "www" in the URL fixed it. (and sorry for sounding condisending in my previous comment. I was frustrated in the moment).
As far as missing some pics when downloading entire albums, make sure you have entered your personal API key from Derpibooru's website. Acting as an anonymous user won't give you all images just like how some images are blocked before you sign in on the actual website. This tool also works for tantabus.ai btw. 😉
Also, check the 'Filter' setting is set to 'Everything' just to ensure it's not that filtering out anything.
